### DISCUSSION TITLE
Timezone of Mirth administrator

### DISCUSSION THREAD

**Discussion by didster** [nextgenhealthcare/connect#4547] [Q&A] [answered]
How does one set the timezone for displayed messages in MC? Its been a while since I've used Mirth but my server is running in a different timezone and I'm sure I used to just do this:

`
javaws -J-Duser.timezone=America/Los_Angeles http://<server>:8080/webstart
`

But that no longer seems to work with the administrator always using my local machine timezone and the only way I can find is to change my local machine timezone which is not an option.

Using MC 3.9.0

Thanks

**Comment by tonygermano** [comment, marked as answer]
Didn't notice until now that you double posted this in the forum, but he's what I just answered there.

> The thing you said used to work doesn't work for me on mirth 3.4
> 
> Setting the jvm option via environment variable did work, though.
> ```
> ~/mirth/launcher/jre/bin$ JAVA_TOOL_OPTIONS=-Duser.timezone=America/Los_Angeles ./javaws http://<server>:8080/webstart
> ```
> I got this from https://stackoverflow.com/questions/40932530/how-to-pass-unsafe-arguments-to-the-jvm-of-a-java-webstart-application
> 
> The accepted answer there said they could not get the -J option passed to javaws to work either.
>

**Reply by didster** [reply]
Sorry about that I got myself confused about which forum is currently being used.

Thanks for the reply, that does indeed work. Messy and ugly, but it works. But hay, I guess that's JavaWS

With this information I then did a shortcut to

C:\Windows\System32\cmd.exe /c "SET _JAVA_OPTIONS=-Duser.timezone=America/Los_Angeles && START "" ^"C:\Program Files\Mirth Connect Administrator Launcher\launcher.exe^"

Which launches the launcher in that timezone and then any MCA's fired up from there are all correct
